### ISSUE TITLE
[Enhancement] Add a wallet description page

### ISSUE THREAD

**Issue by aguillon** [marigold-dev/tzsafe-ui#78]
When clicking on a wallet in the left menu, we're taken to the "Proposals" page, which is most of the time empty. I suggest making a new page, which shows the content of the wallet, the asset it holds (which will make more sense when improving the support for FA tokens) and what not. A nice pie chart would make sense, I guess.

**Comment by rueshyna** [comment]
I agree that. This was discussed before :) . will go this in the near future.
